service/worker/scanner/scanner.go

```diff
@@ -188,7 +193,7 @@ func (s *Scanner) startShardScanner(
 			config.ScannerWFTypeName,
 			shardscanner.NewShardScannerContext(s.context.resource, config),
 		)
-		go workercommon.StartWorkflowWithRetry(
```

Comment:
```go
func StartWorkflowWithRetry(...) error {
	// ...
	sdkClient := client.NewClient(
```
😱 

well.  a lot of this is unavoidable I guess.  we can tackle that mess another time.